### PR TITLE
Added Customer class + CustomerDatabase

### PR DESCRIPTION
**Pull request by MikkelRLarsen**

### CODE REVIEW

File: WinFormsSemesterProjekt/DataBase/CustomerDatabase.cs [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+                );
+
+            command.CommandText = sql;
+            command.Parameters.AddWithValue("@CustomerID", CustomerID);
+
+            using var reader = command.ExecuteReader();
+
+            while (reader.Read())
+            {
+                var ReadCustomer = new Customer(reader.GetInt32(0), reader.GetString(1), reader.GetString(2), reader.GetInt32(3), reader.GetString(4));
```
Comment (SofusOnFire) [inline WinFormsSemesterProjekt/DataBase/CustomerDatabase.cs:58]: The ReadCustomer variable needs to be added to the newly created customer list

File: WinFormsSemesterProjekt/DataBase/CustomerDatabase.cs [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+            command.CommandText = sql;
+            command.Parameters.AddWithValue("@PhoneNumber", phoneNumber);
+
+            using var reader = command.ExecuteReader();
+
+            while (reader.Read())
+            {
+
+                var ReadCustomer = new Customer(reader.GetInt32(0), reader.GetString(1), reader.GetString(2), reader.GetInt32(3), reader.GetString(4));
```
Comment (SofusOnFire) [inline WinFormsSemesterProjekt/DataBase/CustomerDatabase.cs:87]: This variable needs to be added to the newly created customer list too

File: WinFormsSemesterProjekt/DataBase/CustomerDatabase.cs [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+        /// <summary>
+        /// Deletes a customer if given the correct CustomerID of said customer.
+        /// </summary>
+        /// <param name="customer"></param>
+        public static void DeleteCustomer(Customer customer)
+        {
+            SqlCommand command = connection.CreateCommand();
+
+            string sql =
+                "DELETE Customer" +
```
Comment (SofusOnFire) [inline WinFormsSemesterProjekt/DataBase/CustomerDatabase.cs:132]: There needs be a space between Customer and the quotation mark in order for the command to function

File: WinFormsSemesterProjekt/DataBase/CustomerDatabase.cs (hunk 1 above)
Comment (MikkelRLarsen) [reply WinFormsSemesterProjekt/DataBase/CustomerDatabase.cs:58]: Done

**Pull request merged**